api/swagger-spec/v1beta1.json

```diff
@@ -1312,65 +1312,6 @@
     ]
    },
    {
-    "path": "/apis/extensions/v1beta1/namespaces/{namespace}/ingress/{name}/status",
```

Comment:
This is because ingress/status has not been cherrypicked yet.
https://github.com/kubernetes/kubernetes/pull/15783 is pending.